src/analysis/retail/warrior/arms/CONFIG.tsx

```diff
@@ -46,7 +46,8 @@ const config: Config = {
     </>
   ),
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
-  exampleReport: '/report/7wdr9jZN4zBpv2bT/6-Mythic+Shriekwing+-+Kill+(5:36)/Xcyde/standard',
+  exampleReport:
+    'report/B9MWAXq3hFc2znHD/8-Mythic+Terros+-+Kill+(5:28)/Shakeywar/standard/statistics',
```

Comment:
Probably want this to go to the overview instead of statistics.
```suggestion
    'report/B9MWAXq3hFc2znHD/8-Mythic+Terros+-+Kill+(5:28)/Shakeywar/standard',
```